src/qt/locale/bitcoin_de.ts

```diff
@@ -3234,7 +3234,7 @@ Beispiel: alertnotify=echo %%s | mail -s "Bitcoin Alert" admin@foo.com
     </message>
     <message>
         <source>RPC SSL options: (see the Bitcoin Wiki for SSL setup instructions)</source>
-        <translation>RPC-SSL-Optionen (siehe Bitcoin-Wiki für SSL-Einrichtung):</translation>
+        <translation>RPC-SSL-Optionen (siehe Dogecoin-Wiki für SSL-Einrichtung):</translation>
```

Comment:
I think we left that at Bitcoin-Wiki since the other doesn't even have that info (if it even still exists).